crates/bevy_pbr/src/atmosphere/transmittance_lut.wgsl

```diff
@@ -38,10 +38,11 @@ fn ray_optical_depth(r: f32, mu: f32, sample_count: u32) -> vec3<f32> {
 
         let r_i = get_local_r(r, mu, t_i);
 
-        // sampling halfway between the two rows will give the *average*
-        // of absorption and scattering, so need to multiply by 2 to get
-        // the sum.
-        let extinction = sample_density_lut(r_i, 0.5) * 2.0;
+        // PERF: A possible later optimization would be to sample at `component = 0.5`
+        // (getting the average of the two rows) and then multiplying by 2 to find the sum. 
+        let absorption = sample_density_lut(r_i, ABSORPTION_DENSITY);
```

Comment:
Maybe it's better to create two separate functions `sample_absorption` and `sample_density`?